tracing-appender/src/rolling.rs

```diff
@@ -151,6 +152,15 @@ impl RollingFileAppender {
         let writer = RwLock::new(
             create_writer(log_directory, &filename).expect("failed to create appender"),
         );
+
+        // Create a symlink to latest log file.
+        let latest_path = Path::new(&filename);
+        let symlink_path = Path::new(&log_directory).join(&log_filename_prefix);
+        let _ = remove_symlink_file(&symlink_path);
+        if let Err(err) = symlink_file(&latest_path, &symlink_path) {
+            eprintln!("Couldn't create symlink: {}", err);
+        }
```

Comment:
take it or leave it: the code for re-creating the symlink in `Inner::refresh_writer` is identical to this code...I wonder if we want to consider making a function for creating/updating the symlink, and call it in both places?